airflow/providers/common/sql/CHANGELOG.rst

```diff
@@ -24,6 +24,21 @@
 Changelog
 ---------
 
+1.1.1
```

Comment:
wait.. if we are releasing `1.1.1` we have a problem
https://github.com/apache/airflow/pull/25939 set minimum sql provider 1.2.0 for trino/sqlite/presto

Reply:
In this pr I (tried) to change the minimum to those to be 1.1.1

Reply:
Ah you already fixed it :)
resolving this

Reply:
See https://github.com/apache/airflow/pull/26109/files#diff-e1c023651fed5dc86ec9d3303308052dd32afa65a80bfbc6d6c4554d9aae7c1d

Reply:
Nope. Common.sql provider has the new feature (placeholder) so all was fine there. This was a bugfix to trino/presto/sqlite but they dependent on new feature added to common.sql